src/aap_eda/core/enums.py

```diff
@@ -119,6 +119,7 @@ class DefaultCredentialType(DjangoStrEnum):
 # TODO: rename to "RulebookProcessRequest"
 class ActivationRequest(DjangoStrEnum):
     START = "start"
+    RUNNING = "running"
```

Comment:
As extension of the reasoning of [this comment](https://github.com/ansible/eda-server/pull/1088#pullrequestreview-2363157927). "Running" is a state, not a verb. The fact that your are adding such a state here is a clue that the approach might be mistaken.